at1x:
  api_logging: false
  device: "/dev/ttyACM0"
  hardware_interface:
     loop_hz: 100 # hz
     joints:
       - joint1
